The cmake_pkg_config function directly reads pkg-config files, but this is
completely broken in Void's cross-build setup. Use pkg-config directly to avoid
problems.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -112,13 +112,8 @@
 	find_package(unofficial-minizip CONFIG REQUIRED)
 	set(MINIZIP_LIBRARIES "unofficial::minizip::minizip")
 else()
-	if(CMAKE_VERSION VERSION_LESS 3.31)
-		find_package(PkgConfig REQUIRED PkgConfig PkgConf)
-		pkg_check_modules(MINIZIP REQUIRED minizip)
-	else()
-		cmake_pkg_config(EXTRACT minizip REQUIRED)
-		set(MINIZIP_LIBRARIES "minizip")
-	endif()
+	find_package(PkgConfig REQUIRED PkgConfig PkgConf)
+	pkg_check_modules(MINIZIP REQUIRED minizip)
 endif()
 
 # Find the MinGW runtime DLLs.
